library/Kubernetes/Web/MoveFavoriteForm.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/* Icinga for Kubernetes Web | (c) 2025 Icinga GmbH | AGPLv3 */
+
+namespace Icinga\Module\Kubernetes\Web;
+
+use Icinga\Exception\Http\HttpNotFoundException;
+use Icinga\Module\Kubernetes\Common\Auth;
+use Icinga\Module\Kubernetes\Model\Favorite;
+use ipl\Html\Form;
+use ipl\Sql\Connection;
+use ipl\Sql\Expression;
+use ipl\Sql\Select;
+use ipl\Stdlib\Filter;
+use ipl\Web\Common\CsrfCounterMeasure;
+use LogicException;
+use Ramsey\Uuid\Uuid;
+
+class MoveFavoriteForm extends Form
+{
+    use CsrfCounterMeasure;
+
+    protected $defaultAttributes = ['hidden' => true];
+
+    protected $method = 'POST';
+
+    /** @var Connection */
+    protected $db;
+
+    /** @var string */
+    protected $kind;
+
+    /**
+     * Create a new MoveFavoriteForm
+     *
+     * @param ?Connection $db
+     */
+    public function __construct(Connection $db = null)
+    {
+        $this->db = $db;
+    }
+
+    /**
+     * Get the kind
+     *
+     * @return string
+     */
+    public function getKind(): string
+    {
+        if ($this->kind === null) {
+            throw new LogicException('The form must be successfully submitted first');
+        }
+
+        return $this->kind;
+    }
+
+    protected function assemble(): void
+    {
+        $this->addElement('hidden', 'uuid', ['required' => true]);
+        $this->addElement('hidden', 'priority', ['required' => true]);
+    }
+
+    protected function onSuccess(): void
+    {
+        $favoriteUuid = Uuid::fromString($this->getValue('uuid'))->getBytes();
+        $newPriority = $this->getValue('priority');
+
+        /** @var ?Favorite $favorite */
+        $favorite = Favorite::on($this->db)
+            ->columns(['kind', 'priority'])
+            ->filter(Filter::all(
+                Filter::equal('resource_uuid', $favoriteUuid),
+                Filter::equal('username', Auth::getInstance()->getUser()->getUsername())
+            ))
+            ->first();
+        if ($favorite === null) {
+            throw new HttpNotFoundException('Favorite not found');
+        }
+
+        $transactionStarted = ! $this->db->inTransaction();
```

Comment:
I just want to leave a comment to double check this change since we already have a comment for code that does the same.